feature_appview/src/main/java/cm/aptoide/pt/feature_appview/presentation/AppViewView.kt

```diff
@@ -0,0 +1,682 @@
+    modifier = Modifier
+      .fillMaxWidth()
+      .fillMaxHeight()
+    //.padding(bottom = 65.dp)
+  ) {
+    if (!uiState.isLoading) {
+      uiState.app?.let {
+        AppViewContent(
+          app = it,
+          selectedTab = uiState.selectedTab,
+          tabsList = uiState.tabsList,
+          similarAppsList = uiState.similarAppsList,
+          similarAppcAppsList = uiState.similarAppcAppsList,
+          otherVersionsList = uiState.otherVersionsList,
+          relatedContentList = uiState.relatedContent,
+          onSelectTab = onSelectTab, onSelectReportApp
+        )
+        onFinishedLoadingContent(it.packageName)
+      }
+
+    }
+  }
+}
+
+@Composable
+fun AppViewContent(
+  app: App,
+  selectedTab: AppViewTab,
+  tabsList: List<AppViewTab>,
+  similarAppsList: List<App>,
+  similarAppcAppsList: List<App>,
+  otherVersionsList: List<App>,
+  relatedContentList: List<RelatedCard>,
+  onSelectTab: (AppViewTab) -> Unit,
+  onSelectReportApp: (App) -> Unit,
+) {
+  LazyColumn(
+    modifier = Modifier
+      .fillMaxSize()
+      .padding(bottom = 65.dp)
+  ) {
+    val listScope = this
+
+    item {
+      Image(
+        painter = rememberImagePainter(app.featureGraphic,
+          builder = {
+            placeholder(cm.aptoide.pt.feature_apps.R.drawable.ic_placeholder)
+            transformations(RoundedCornersTransformation())
+          }),
+        contentDescription = "App Feature Graphic",
+        modifier = Modifier
+          .fillMaxWidth()
+          .height(181.dp)
+          .padding(bottom = 19.dp)
+      )
+    }
+
+    item {
+      AppPresentationView(app)
+    }
+    item {
+      AppStatsView(app)
+    }
+    item {
+      InstallButton(app)
+    }
+
+    item {
+      AppInfoViewPager(
+        selectedTab,
+        tabsList,
+        onSelectTab
+      )
+    }
+
+    item {
+      ViewPagerContent(
+        app,
+        selectedTab,
+        similarAppsList,
+        similarAppcAppsList,
+        otherVersionsList,
+        relatedContentList,
+        onSelectReportApp,
+        listScope
+      )
+    }
+  }
+}
+
+@Composable
+fun AppInfoViewPager(
+  selectedTab: AppViewTab,
+  tabsList: List<AppViewTab>,
+  onSelectTab: (AppViewTab) -> Unit
+) {
+  Column(
+    modifier = Modifier
+      .fillMaxWidth()
+      .padding(top = 42.dp, start = 16.dp, end = 16.dp)
+  ) {
+    ScrollableTabRow(
+      selectedTabIndex = selectedTab.index,
+      contentColor = Color(0xFFFE6446),
+      backgroundColor = Color.Transparent,
+      modifier = Modifier
+        .fillMaxWidth(), edgePadding = 0.dp
+    ) {
+      tabsList.forEachIndexed { index, tab ->
+        Tab(
+          selected = selectedTab == tab,
+          onClick = { onSelectTab(tab) },
+        ) {
+          Text(
+            text = tab.tabName,
+            modifier = Modifier.padding(bottom = 12.dp, end = 22.dp),
+            fontSize = MaterialTheme.typography.subtitle1.fontSize
+          )
+        }
+      }
+    }
+  }
```

Comment:
tab title needs to be centered, else both underline and long press is offsetted.